source/develop/release-management/features/network/ipv6_subnets.html.md

```diff
@@ -65,32 +106,17 @@ IPv6 provides the following IPAM options:
 
-* dhcpv6-stateless: IP addresses are assigned by OVN's IPv6 stack, based on the port's mac address and the advertised prefix - in the RA messages.The connection parameters (MTU, max hops, DNS, etc) are configured through the subnet's native DHCPv6 service.
-
-* dhcpv6-stateful: Both the IP addresses and the connection parameters are assigned from the subnet's native DHCPv6 service.
+* dhcpv6-stateless: IP addresses are assigned by OVN's IPv6 stack, based on the port's mac address and the advertised prefix - in the RA messages. The connection parameters (MTU, max hops, DNS, etc) are configured through the subnet's native DHCPv6 service.
 
-Take into account that only the stateful DHCPv6 IPAM type is in this feature's scope.
+* dhcpv6-stateful: Both the IP addresses and the connection parameters are assigned from the subnet's native DHCPv6 service. DHCPv6 protocol does not send the network prefix in the **Reply** messages, which means that this configuration will only work if the network's prefix length is 64.
 
 ### IPAM configuration requirements
 
 * slaac: this IPAM configuration does not rely on OVN's subnet DHCP service, which means a DHCP_Options object is **not** required. The RA messages are sent by an OVN Logical_Router, thus one is required.
 
 * dhcpv6-stateless: this IPAM configuration relies on OVN's subnet DHCP services to get its IP address, and on the Logical_Router's RA message's connection parameters to properly configure the connection. Thus, a DHCP_Options entry is required, as is a Logical_Router object.
 
-* dhcpv6-stateful: this IPAM configuration relies exclusively on OVN's subnet DHCP services, and is very close to the IPv4 configuration counterpart - a DHCP_Options object is required, having both an IPv6 cidr, and ip_version set to IPv6.
-
-### IPAM configuration
-
-This section explains how to configure the supported IPAM options.
-
-* dhcpv6-stateful
-  - create an ovn DHCP_Options object, having an ipv6 cidr configured.
-    It requires the user to specify the ethernet address for the DHCPserver
-    to use in the *options:server_id* DHCP_Options attribute.
-  - to enable IPAM for IPv6, the logical switch (ovn entity) where the ports are
-    created has to include the **other_config: ipv6_prefix** key defined.
-    The value should be the the IP prefix featured in the provisioned cidr
-    parameter, ending in '::' - e.g. 8230:5678::, for instance.
+* dhcpv6-stateful: this IPAM configuration relies exclusively on OVN's subnet DHCP services, and is very close to the IPv4 configuration counterpart - a DHCP_Options object is required, having both an IPv6 cidr, and ip_version set to IPv6. On newer versions of dhclient, a router is also required, since it the dhcpv6 messages **do not** feature the network prefix length.
```

Comment:
since it the dhcpv6 messages -> since the dhcpv6 messages

or
since the dhcpv6 messages ... anymore
as they no longer feature ....

Reply:
Done